src/connect_wrap-inl.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SRC_CONNECT_WRAP_INL_H_
+#define SRC_CONNECT_WRAP_INL_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "connect_wrap.h"
+#include "util.h"
+#include "util-inl.h"
+#include "v8.h"
+
+namespace node {
+
+using v8::FunctionCallbackInfo;
+using v8::Value;
+
+static inline void NewConnectWrap(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Two nits:

1. Can you use fully qualified names in headers, e.g., `v8::FunctionCallbackInfo<v8::Value>`?
2. `static inline` is tautological (in C++), just `inline` is sufficient. :-)